swift-sdk/IterableInboxViewController.swift

```diff
@@ -433,8 +436,14 @@ extension IterableInboxViewController: InboxViewControllerViewModelView {
     
     private func updateUnreadBadgeCount() {
         let unreadCount = viewModel.unreadCount
-        let badgeValue = unreadCount == 0 ? nil : "\(unreadCount)"
-        navigationController?.tabBarItem?.badgeValue = badgeValue
+        let badgeUnreadCount = unreadCount == 0 ? nil : "\(unreadCount)"
+        let badgeUnreadIndicator = unreadCount == 0 ? nil : ""
+        
+        if showCountInUnreadBadge {
+            navigationController?.tabBarItem?.badgeValue = badgeUnreadCount
+        } else {
+            navigationController?.tabBarItem?.badgeValue = badgeUnreadIndicator
```

Comment:
This should be nil. The customer does not want to show either a "dot or a badge". This will probably show a dot.